src/controllers/document_upload/ajax.upload.responder.strategy.ts

```diff
@@ -62,8 +62,8 @@ export class AjaxUploadResponderStrategy implements UploadResponderStrategy {
    * @param {Attachment[]} attachments list of uploaded attachments
    */
   public handleGovUKError = async (res: Response,
-                                   errorData: GovUkErrorData,
-                                   attachments: Attachment[]) => {
+    errorData: GovUkErrorData,
```

Comment:
You need to migrate the tslint rules that we applied over to eslint, there was a rule for param alignment.

Reply:
Migrated the rules - names and definitons very different.